.github/actions/deploy-docs.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
IMHO as for file name, think instead "docs" use "storybook"
It will be clear what kind of content will be deployed.